repository/build.gradle

```diff
@@ -6,6 +6,7 @@ dependencies {
     implementation group: 'org.springframework', name: 'spring-jdbc', version: springVersion
     implementation group: 'org.springframework', name: 'spring-context', version: springVersion
     implementation group: 'org.springframework', name: 'spring-beans', version: springVersion
+    implementation group: 'org.springframework', name: 'spring-orm', version: springHibernate

```

Comment:
Why do you need this dependency?

Reply:
To initialize an entity manager bean (i.e. one of the imports necessary `import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;`)

Reply:
You can do without this Bean, as far as I know.

Reply:
Have tried it, with no success
![image](https://user-images.githubusercontent.com/56612094/140039798-ab3eae06-30c7-41c9-83a0-cbf8257dd70c.png)